src/awsexplorer/activation.ts

```diff
@@ -53,9 +53,11 @@ export async function activate(activateArguments: {
         activateArguments.regionProvider
     )
 
-    activateArguments.context.subscriptions.push(
-        vscode.window.registerTreeDataProvider(awsExplorer.viewProviderId, awsExplorer)
-    )
+    const view = vscode.window.createTreeView(awsExplorer.viewProviderId, {
```

Comment:
Are we using this const elsewhere?

Reply:
No but discovering that these `view` objects can be listened-to is non-obvious, so it's worth leaving as a hint...